cmd/zed/zed.d/io-spare.sh

```diff
@@ -209,22 +177,21 @@ main()
         #
         # Round-robin through the spares trying those that are available.
         #
-        for spare in ${ZEVENT_VDEV_SPARE_PATHS}; do
-
-            # shellcheck disable=SC2046
-            set -- $(query_vdev_status "${ZEVENT_POOL}" "${spare}")
-            spare_path="$1"
-            spare_status="$2"
```

Comment:
Oh, I moved this to the <code>get_pool_spares()</code> function instead. Didn't make much sense in returning failed spare from that function, and THEN check if it's available.